build.sbt

```diff
@@ -1,28 +1,8 @@
 import sbt._
 
-// This supports a kind of cross-building for Scalding 0.12.0 and 0.15.0
-// The default is 0.12.0, but adding the -Dscalding.version=0.15 argument on the command line
-// will cause this to build with 0.15 instead
-val scaldingVersionProp = System.getProperty("scalding.version") match {
```

Comment:
This kind of cross building isn't feasible to support in the long term so I'm taking it out now.